src/chisel/lib/uart/Uart.scala

```diff
@@ -10,13 +10,14 @@
 package chisel.lib.uart
 
 import chisel3._
+import chisel3.experimental.Analog
 import chisel3.util._
+import fpga.ip.ice40._
```

Comment:
This file should not depend on the specific FPGA. It should be a library that can be used by a variety of different FPGAs.